api/configs/feature/__init__.py

```diff
@@ -150,7 +150,7 @@ class CodeExecutionSandboxConfig(BaseSettings):
 
     CODE_MAX_STRING_LENGTH: PositiveInt = Field(
         description="Maximum allowed length for strings in code execution",
-        default=80000,
+        default=400000,
```

Comment:
```suggestion
        default=400_000,
```
Minor suggestion. Follow the similar syle setting for `TEMPLATE_TRANSFORM_MAX_LENGTH` as well.